client/components/app-promo/lib/promo-retriever.js

```diff
@@ -0,0 +1,35 @@
+const promo_options = [
+	{
+		promo_code: 'a0001',
+		message: 'WordPress.com your way  — desktop app now available for Mac, Windows, and Linux.'
+	},
+	{
+		promo_code: 'a0002',
+		message: 'Get WordPress.com app for your desktop.'
+	},
+	{
+		promo_code: 'a0003',
+		message: 'WordPress.com app now available for desktop.'
+	},
+	{
+		promo_code: 'a0005',
+		message: 'WordPress.com at your fingertips — download app for desktop.'
+	},
+	{
+		promo_code: 'a0006',
+		message: 'WordPress.com in the palm of your hands — download app for mobile.'
+	}
+];
+
+export const getRandomPromo = ( ) => {
+	return promo_options[ Math.floor( Math.random() * promo_options.length ) ];
```

Comment:
checkout `sample` from **lodash** - you might find it useful here!

```js
import { sample } from 'lodash';

export const getRandomPromo = () => sample( promo_options );
```

also, we don't put a space in the `( )`